AIPscan/Aggregator/views.py

```diff
@@ -317,3 +323,27 @@ def get_mets_task_status(coordinatorid):
     response = {"state": "COMPLETED"}
     flash("Fetch Job {} completed".format(downloadStart))
     return jsonify(response)
+
+
+@aggregator.route("/index_refresh/<fetch_job_id>")
+def index_refresh(fetch_job_id):
+    tasks.start_index_task(fetch_job_id)
```

Comment:
Shouldn't be checking if Typesense is enabled here or in the JS code before triggering the indexing task?

Reply:
Done!